im2latex/trainers/visionencoderdecodertrainer.py

```diff
@@ -0,0 +1,332 @@
+from loguru import logger
+from torch.nn.parallel import DistributedDataParallel as DDP

```

Comment:
Are we planning to distribute the training? I tried using this process on my PC and had to setup some environment variables and software to get it to work. It does not make sense since we only have a single GPU.

Reply:
My idea was to keep the distributed functionality in the code since the original codebase had it and then have functionality that allows us to flexibly switch to the single GPU case.